### ISSUE TITLE
Colour contrast issues with alpha warning

### ISSUE THREAD

**Issue by MelissaAutumn** [thunderbird/stormbox#46]
The alpha warning fails [WCAG](https://www.w3.org/WAI/WCAG22/Understanding/contrast-minimum) check. 

Maybe make the text colour black inside that yellow warning box.

**Comment by Sancus** [comment]
I think all my colors will change a bit when I pull in services ui, hopefully it has some kind of warning banner component that I can use?

**Comment by alt-aly-cat** [comment]
I mentioned this in matrix as well, but I think it got lost or I forgot to follow up again. I was wondering if we could use [Primary button](https://thunderbird.github.io/services-ui/?path=/story/components-button--variant) and [Banner](https://thunderbird.github.io/services-ui/?path=/docs/components-noticebar--docs) so that it passes accessibility. We also mocked it up in [Figma.](https://www.figma.com/design/6Xn9ltELZMhY5wXPBzlayX/tMail_web_2026-Q2_tMailFirst_v02?node-id=3463-23958&t=Pwye8WgP4bf0kV0B-1) I can push to Zeplin if that's helpful?

**Comment by Sancus** [comment]
> I mentioned this in matrix as well, but I think it got lost or I forgot to follow up again. I was wondering if we could use [Primary button](https://thunderbird.github.io/services-ui/?path=/story/components-button--variant) and [Banner](https://thunderbird.github.io/services-ui/?path=/docs/components-noticebar--docs) so that it passes accessibility. We also mocked it up in [Figma.](https://www.figma.com/design/6Xn9ltELZMhY5wXPBzlayX/tMail_web_2026-Q2_tMailFirst_v02?node-id=3463-23958&t=Pwye8WgP4bf0kV0B-1) I can push to Zeplin if that's helpful?

Yeah when I pull in services-ui it should use the button component for all the buttons in the app and that banner component in warning type looks good to me so I'll use that.
